pkg/packagekit/updates.jsx

```diff
@@ -275,8 +275,8 @@ class UpdateItem extends React.Component {
         }
 
         var pkgList = this.props.pkgNames.map(n => (
-            <OverlayTrigger key={n} overlay={ <Tooltip id="tip-summary">{packageSummaries[n]}</Tooltip> } placement="top">
-                <span>{n}</span>
+            <OverlayTrigger key={n.name + n.arch} overlay={ <Tooltip id="tip-summary">{packageSummaries[n.name] + " for " + n.arch}</Tooltip> } placement="top">
```

Comment:
Let's please avoid the "for", it needs to be translated. How about just showing it in parentheses?